defender/gasless-nft-minting/README.md

```diff
@@ -0,0 +1,168 @@
+
+## Setup
+
+This section will help you set up the following:
+- Acquire Defender and Relay keys
+- Add funds to the Relay
+- Deploy contracts
+  - `yarn deploy contract --contract-name MinimalForwarder`
+  - `yarn deploy contract --contract-name DemoNFT "0xMINIMAL_FORWARDER_ADDRESS"`
+- Update `config.dev.yml` with the 2 deployed contract addresses
+- Deploy the Autotasks with `serverless deploy`
+- Connect Relays to deployed Autotasks
+  - Contract Deployer relay must be connected to the signing Autotask
+  - Any Relayer with funds can be connected to the Relay Autotask
+- Copy the 2 Autotask webhooks into the `config.dev.yml` file
+
+### Defender Account Setup
+
+- In your [Defender account](https://defender.openzeppelin.com/), select the Hamburger icon in the upper right corner and click on **Team API Keys**
+- In the Team API Keys screen, click **Create API Key**
+- Make sure that the options for **Manage Autotasks** and **Manage Proposals and Contracts** are selected (we do not make use of the **Manage Relayers** and **Manage Sentinels** options with this template)
+- Click **Save**
+- Copy your API key and Secret key to a local file (you will **NOT** be able to view your Secret key again after this message box goes away)
+- Make sure that you really did copy your API key and Secret key to a local file
+- Check the box for **I’ve written down the Secret key** and select **Close**
+
+### Relay API Keys
+
+- In your [Defender account](https://defender.openzeppelin.com/), select the **◎ Relay** link on the upper left side to get to the Relay Dashboard
+- If you do not have a Relay on the network that you want to deploy to, create one:
+  - In the Relay Dashboard screen, click **Create Relayer** in the upper right corner
+  - Pick a meaningful name, choose the network that you want to deploy to, and click **Create**
+- Click on the existing Relay that you would like to use for deployments
+- Click on the settings gear/cog **⚙** at the top middle and choose **+ Create new API key**
+- Copy your Ethereum Address, API key, and Secret key to a local file (you will **NOT** be able to view your Secret key again after this message box goes away)
+- Make sure that you really did copy your API key and Secret key to a local file
+- Check the box for **I’ve written down the secret key** and select **Close**
+
+### Add Funds to the Relay
+
+- In your [Defender account](https://defender.openzeppelin.com/)
+- Click on the **Relay** link on the left side
+- Find the Relay that deployed the NFT contract, click on the **copy address** button
+- Transfer funds to that address to cover the deployment and NFT minting costs
+
+### Local Code Setup
+
+In the `gasless-nft-minting` directory, perform the following steps:
+
+- A secrets file can be created for each stage of production. We will create one for development:
+  - Copy and rename the `sample.secrets.yml` to `.secrets/dev.yml`
```

Comment:
The `sample.secrets.yml` file is not in the `gasless-nft-minting` directory. It is in the `defender` directory.

Reply:
Fixed

Reply:
9daa9144369044130f497f8c4a41219aab381ec9